doc/content/verification_validation_examples/plasma/lieberman.md

```diff
@@ -0,0 +1,175 @@
+
+### Kernels
+
+Each variable only needs a single kernel. The electrostatic potential, `phi`, requires this diffusion kernel which enables the system to solve Poisson's equation.
+
+!alert note title=
+Since FENIX does not apply the factor $\varepsilon_0^{-1}$ directly when evalulating the inner product of the particle charge density and the test function the quantity $\varepsilon_0$ is shifted to the laplacian operator.
+
+The variable `n` utilizes the [ProjectionKernel.md] to enable visualization of the particle number density as a field variable. See [ProjectionKernel.md] for more detail.
+
+!listing test/tests/benchmarking/lieberman.i block=Kernels
+
+### Boundary Conditions
+
+The only variable that needs a boundary condition is the electrostatic potential. Since our simulation assumes that end points of the domain are grounded.
+
+!listing test/tests/benchmarking/lieberman.i block=BCs
+
+### Field Initial Condition
+
+The only variable which needs an initial condition is the electrostatic potential, here we prescribe the initial condition of the electrostatic potential to be the analytic solution for the electrostatic potential given the prescribed uniform charge density.
+
+!listing test/tests/benchmarking/lieberman.i block=Functions ICs
+
+### AuxVariables
+
+Here one auxilary variable is created for each field component. We require three here since our particles always have 3 velocity components stored in data.
+
+!listing test/tests/benchmarking/lieberman.i block=AuxVariables
+
+### AuxKernels
+
+For this simulation only the x component of the electric field is being used in this block the electric field is computed from the electrostatic potential. The other AuxVariables do not have any kernels associated with them and as a result they will have a value of 0 throughout the simulation.
+
+!listing test/tests/benchmarking/lieberman.i block=AuxKernels
+
+### Particle Initialization and Updating
+
+The `UserObjects` block is where particles are created and the rules for how the particle velocity is update by the fields are defined. The `stepper` is a [LeapFrogStepper.md] which updates a particles' velocity based on the value of the electric fields at the location at which the particle exists. The `initializer` defines the rules for how particles are placed in the mesh and the how their velocities are initialized. The [UniformGridParticleInitializer.md] places particles evenly throughout the mesh. In this case 100 particles are placed on the mesh with uniform spacing between them and they are weighted so that this particle distribution will approximate the specified argon ion number density, `n`, the last parameter `velocity_distributions` tells the system which distributions that you would like to sample from when initializing particle velocity data. The final object is the `TestInitializedPICStudy` this object and anyother objects which inherit from [PICStudyBase.md] are responsible for managing the particles.
```

Comment:
```suggestion
The `UserObjects` block is where particles are created and the rules for how the particle velocity is updated by the fields are defined. The `stepper` is a [LeapFrogStepper.md] which updates a particles' velocity based on the value of the electric fields at the location at which the particle exists. The `initializer` defines the rules for how particles are placed in the mesh and the how their velocities are initialized. The [UniformGridParticleInitializer.md] places particles evenly throughout the mesh. In this case, 100 particles are placed on the mesh with uniform spacing between them, and they are weighted so that this particle distribution will approximate the specified argon ion number density, `n`. The last parameter, `velocity_distributions`, tells the system which distributions that should be sampled from when initializing particle velocity data. The final object is the `TestInitializedPICStudy`. This object and any other objects which inherit from [PICStudyBase.md] are responsible for managing the particles themselves.
```